app/assets/stylesheets/govuk_publishing_components/components/govspeak/_typography.scss

```diff
@@ -11,6 +11,8 @@
 .gem-c-govspeak {
   @include govuk-text-colour;
 
+  @include core-16;
```

Comment:
`@include govuk-font($size: 16);`?

Reply:
I didn't do that then as I don't want to have sections/files where we have some govuk-frontend font mixins and some govuk_frontend_toolkit ones. It makes it look like they have special meaning rather than one legacy and one not.

Really I should have just stopped at that comment and said "no" but no I had to had try be a hero and I've converted all the govspeak font and media queries to use govuk-frontend mixins. I've put this in as a separate commit so it's easy to chop off this if it looks problematic.

I've put together a diff that shows how the resulting CSS changes if you want to have a scan of that and see if that introduces any areas of concern: https://www.diffchecker.com/1IHtaMg8

Reply:
Wow.. you really went down the 'hero' road. I'll do some testing, but looks great so far.

Reply:
All good, I think we can get this merged.

Reply:
Great 👍 - before we merge do you want to see if there's anything we can about the barchart selector here?

Reply:
Seems to be working fine as it is. I'd like to have a look at the `magnaCharta` repo and update the script to prevent is from being initialised twice (which was a good find-out) - I made a note to look into it.